backend/utils/generate_mock_data.js

```diff
@@ -36,23 +39,34 @@ const addSampleResource = resource => {
   return newResource.save();
 };
 
+const addSampleUser = user => {
+  const newUser = new User(user);
+  return newUser.save();
+};
+
 const main = async () => {
   const args = process.argv.slice(2);
   const resourceCountLimit = args[0];
   const shouldUseLoremData = args[1] === "lorem";
   await createConnection();
   console.log(colors.green("Clearing all existing resource data..."));
   await Resource.deleteMany({});
-  const data = shouldUseLoremData ? await fetchJson() : fetchFromFile();
+  const data = shouldUseLoremData
+    ? await fetchJson(JSON_LINK_RESOURCES)
+    : fetchFromFile();
+  const userData = await fetchJson(JSON_LINK_USERS);
   const resources = resourceCountLimit
     ? data.slice(0, parseInt(args[0])).map(addSampleResource)
     : data.map(addSampleResource);
+  const users = userData.map(addSampleUser);
```

Comment:
This indiscriminately adds mock users, even in prod. Oop

Reply:
Oop, and I take full responsibility for allowing this to have affected our shared DB before it was merged. It was a good catch.